### DISCUSSION TITLE
How can non-uniform gridded data assigned to constant parameter in Wflow_sbm?

### DISCUSSION THREAD

**Discussion by Safa136** [Deltares/Wflow.jl#335] [Q&A] [answered]
What methods can be employed to input non-uniform gridded data, such as spatially varying values of a parameter like Ksathorfrac, into Wflow for model calibration?

**Comment by verseve** [comment, marked as answer]
Often these parameters are calibrated against station discharge data, in combination with other (sensitive) model parameters, based on an algorithm of your choice (see for example the Python package SPOTPY). For `KsatHorFrac` there is also this open [issue](https://github.com/Deltares/hydromt_wflow/issues/228) in hydromt_wflow that could be of interest. Another technique that could be of interest is [Multiscale Parameter Regionalization](https://www.ufz.de/index.php?en=40126).

**Reply by Safa136** [reply]
Thank you for your reply.
